public/assets/styles/login.scss

```diff
@@ -0,0 +1,121 @@
+@import url('https://fonts.googleapis.com/css?family=Zilla+Slab');
+@import url('https://fonts.googleapis.com/css?family=Open Sans');
```

Comment:
Nice, but these two import links should be one.

Reply:
@KvNGCzA Thanks. its done